src/cljs/rems/application.cljs

```diff
@@ -680,12 +680,25 @@
                   [application-link (:application/copied-to event) (text :t.applications/application)]
 
                   (not-empty (:application/comment event)))
-        time (localize-time (:event/time event))]
+        time (localize-time (:event/time event))
+        event-public (:application/public event)]
     [:div.row.event
      [:label.col-sm-2.col-form-label time]
      [:div.col-sm-10
       [:div.event-description.d-flex.justify-content-between.col-form-label
-       [:b.col.pl-0 event-text]]
+       [:div.col.pl-0
+        (cond
+          (and see-everything
```

Comment:
Do we really need to check `see-everything`? If the user doesn't see the event, it's not even sent from the API. For the handling side, where we want to show the publicity, we could include a precalculated publicity property for every event like `:event/public`?